Minor wordsmithing (#104)
network-quality/draft-ietf-ippm-responsiveness: draft-ietf-ippm-responsiveness.md

For many years, a lack of responsiveness, variously called
lag, latency, or bufferbloat, has been recognized
as an unfortunate, but common, symptom in today's networks {{Bufferbloat}}.
Solutions like fq_codel {{RFC8290}}, PIE {{RFC8033}}, Cake {{Cake}}
and L4S {{RFC9330}} have been standardized

> and are to some extent implemented.

Reviewer: ```suggestion
and are, to some extent, implemented.
```

Author: Fixed.

tradeoff between using realistic traffic patterns and pushing the network to
its limits.

The Responsiveness Test defines working conditions as the condition
where the path between the measuring endpoints is fully utilized at
its end-to-end capacity, and consequently,
if the sending rate increases even a little, a queue will begin to build

> up at the entry to the bottleneck link. How the bottleneck device

Reviewer: ```suggestion
up at the ingress to the bottleneck hop. How the device at the ingress to the bottleneck hop
```

A _absolute_ suggestion -- I just thought it would be worth trying continue the use of "hop".

Author: Fixed.

other parameters of the methodology and the network-environment it may be that a
measurement never converges to a stable point.
This is expected and part of the dynamic nature of networking and the accompanying
measurement inaccuracies, which is why the importance of imposing a time-limit
is so crucial, together with an accurate depiction of the "confidence" the methodology
was able to generate. The confidence score should be reported to the user as part of
the main results.

# Interpreting responsiveness results

The described methodology uses a high-level approach to measure responsiveness.

> By executing the test with normal HTTP requests a number of elements come into

Reviewer: ```suggestion
By executing the test with normal HTTP requests, a number of elements come into
```

Author: Fixed.

by the server selection. Second, the network stack deployed on the selected server
may not be representative for workloads that are relevant to the user running the test.

## Investigating Poor Responsiveness

> Once a responsiveness result has been generated one might be motivated to try to localize

Reviewer: ```suggestion
Once a responsiveness result has been generated, one might be motivated to try to localize
```

Author: Fixed.
